test/dummies/vars.go

```diff
@@ -160,6 +169,15 @@ func SetDummyCSMachineVars() {
 			Offering: capcv1.CloudStackResourceIdentifier{
 				Name: "Offering",
 			},
+			DiskOffering: capcv1.CloudStackResourceDiskOffering{
+				CloudStackResourceIdentifier: capcv1.CloudStackResourceIdentifier{
+					Name: "DiskOffering",
+				},
+				MountPath:  "/data",
+				Device:     "/dev/vdb",
+				Filesystem: "ext4",
+				Label:      "data_disk",
```

Comment:
Should we have more validation on these fields to make sure they're valid? At least to make sure that "filesystem" is properly input as a courtesy to users who may have made a typo?

Reply:
those fields are not used by CAPC directly and verification logic is inside eks-a. See above comments for background.